code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -537,7 +537,7 @@
 	icon_state = "swat2helm"
 	item_state = "swat2helm"
 	desc = "A tactical SWAT helmet MK.II."
-	armor = list("melee" = 40, "bullet" = 50, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 40, "bullet" = 50, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 40, "fire" = 80, "acid" = 100)
```

Comment:
ditto

Reply:
i don't even remember doing the rad part, but i just said why on the fire thing